ynput/ayon-core
# Houdini: instance in current context validator

## Changelog Description
Add optional validator check on invalid context data for asset and task in Houdini.

Many thanks to BigRoy https://github.com/ynput/ayon-core/issues/162#issuecomment-1985653191

## Testing notes:
1. Launch Houdini.
2. Create any instance and change its context to another asset/shot.
3. `ValidateInstanceInContextHoudini` should raise error and offer a repair action.
4. Click Repair Action, and it should work fine. 
4. This validator only error if there are any instances are pointing to different asset than the current.
